server/views/components/image/index.njk

```diff
@@ -1,23 +1,24 @@
 <figure class="figure
                {{ ' figure--full' if full }}
                {{ ' figure--bleed' if bleed }}
-               {{ ' figure--max-out-height' if maxOutHeight }}">
+               {{ ' figure--max-out-height' if maxOutHeight }}
+               {{ ' image-gallery__figure' if galleryImage }}">
```

Comment:
This is very specific t the image gallery, thus negating the abstraction really.

I wonder if we could abstract out the `<picture>` element, that could then take classes (or even more abstract attributes) from this component and the gallery component?

Then we can just override the `block figcaption`.

I ask as adding loads of things and dependencies to one `method`(i.e. `component`) makes it more complex and rife for bugs to appear.